python/core/auto_generated/qgsdataprovider.sip.in

```diff
@@ -265,8 +265,11 @@ databases and servers.
 
     virtual void reloadData();
 %Docstring
-Reloads the data from the source. Needs to be implemented by providers with data caches to
-synchronize with changes in the data source
+Reloads the data from the source by calling reloadProviderData() implemented
+by providers with data caches to synchronize, changes in the data source, feature
+counts and other specific actions and it calls the dataChanged signal
```

Comment:
```suggestion
counts and other specific actions and it.
Emits the `dataChanged` signal
```

To make it clearer that this is done by this method and not by subclasses.

Reply:
Should we use markdown `` for method names in the docs?

Reply:
I think so.
Or even better something like "Emits the QgsDataProvider::dataChanged() signal" to autolink
http://www.doxygen.nl/manual/autolink.html